src/Common/Commands.Common/AzureDataCmdlet.cs

```diff
@@ -111,48 +111,17 @@ protected override void SaveDataCollectionProfile()
             WriteWarning(string.Format(Resources.DataCollectionSaveFileInformation, fileFullPath));
         }
 
-        protected override void PromptForDataCollectionProfileIfNotExists()
+        protected override void SetDataCollectionProfileIfNotExists()
         {
-            // Initialize it from the environment variable or profile file.
             InitializeDataCollectionProfile();
 
-            if (!_dataCollectionProfile.EnableAzureDataCollection.HasValue && CheckIfInteractive())
```

Comment:
@twitchax this change looks huge + no tests.
What is the plan to test this change out, what are the various scenario should we test this on?
Can we list out those scenarios?